### PR TITLE
Studio: Rename 'force import' to 're-import'

### PR DESCRIPTION
**Pull request by jellebouwman**
Counterpart of https://github.com/iterative/studio/pull/7828

### CODE REVIEW

**Comment by jellebouwman** [comment]
@tapadipti since https://github.com/iterative/studio/pull/7828#event-10532444696 got merged, should we ship this one too?

File: content/docs/studio/user-guide/team-collaboration/index.md [hunk 1]
```diff
@@ -7,12 +7,21 @@ team.
 
 In this page, you will learn about:
 
-- [How to create a team](#create-a-team)
-- [How to invite collaborators (team members)](#invite-collaborators)
-- [The privileges (access permissions) of different roles](#roles)
-- [How to manage the team and its projects](#manage-your-team-and-its-projects)
-- [How to manage connections to self-hosted GitLab servers](#manage-connections-to-self-hosted-gitlab-servers)
-- [Why and how to change your team plan and team size](#change-your-team-plan-and-team-size)
+- [Teams](#teams)
```
Comment (tapadipti) [inline content/docs/studio/user-guide/team-collaboration/index.md:10]: @jellebouwman Is there a particular reason to make this change? I feel like it was better before. If you still want to go ahead with this change, can we make it into a separate PR and limit the current PR to `force import` changes?

File: content/docs/studio/user-guide/troubleshooting.md [hunk 2]
```diff
@@ -13,25 +13,35 @@ If you need further help, you can send us a message using `Help` on the
...
+  - [Error: Base commit not found](#error-base-commit-not-found)
+  - [Error: Failed to push experiment to repository](#error-failed-to-push-experiment-to-repository)
+  - [Project does not display live metrics and plots](#project-does-not-display-live-metrics-and-plots)
+  - [Project does not display DVC experiments](#project-does-not-display-dvc-experiments)
+  - [Error: `dvc.lock` validation failed](#error-dvclock-validation-failed)
+  - [Project does not reflect updates in the Git repository](#project-does-not-reflect-updates-in-the-git-repository)
+  - [I cannot find my desired Git repository in the form to add a model](#i-cannot-find-my-desired-git-repository-in-the-form-to-add-a-model)
+  - [Model registry does not display the models in my Git repositories](#model-registry-does-not-display-the-models-in-my-git-repositories)
+  - [My models have disappeared even though I did not remove (deprecate) them](#my-models-have-disappeared-even-though-i-did-not-remove-deprecate-them)
+  - [Questions or problems with billing and payment](#questions-or-problems-with-billing-and-payment)
```
Comment (tapadipti) [inline content/docs/studio/user-guide/troubleshooting.md:44]: Same as before - Is there a particular reason to make this change? I feel like it was better before. If you still want to go ahead with this change, can we make it into a separate PR and limit the current PR to `force import` changes?

**Comment by tapadipti** [comment]
Thanks for updating the docs @jellebouwman. I think it's better to limit this PR to the `force import` changes. Let me know if you think otherwise.

File: content/docs/studio/user-guide/troubleshooting.md (hunk 2 above)
Comment (jellebouwman) [reply content/docs/studio/user-guide/troubleshooting.md]: Sorry @tapadipti - my IDE seems to have included these changes without me noticing.

File: content/docs/studio/user-guide/team-collaboration/index.md (hunk 1 above)
Comment (jellebouwman) [reply content/docs/studio/user-guide/team-collaboration/index.md]: Sorry @tapadipti - my IDE seems to have included these changes without me noticing.

**Review by tapadipti — APPROVED** [review]
